### ISSUE TITLE
[vite-node] [ERR_INVALID_URL_SCHEME] /.nuxt/i18n.options.mjs

### ISSUE THREAD

**Issue by m0zz094** [nuxt-modules/i18n#1581]
### Version
@nuxtjs/i18n: ^8.0.0-beta.1
nuxt: 3.0.0-rc.12

### Nuxt configuration

Please change to `[x]` if relevant for this issue:

 - [ ] Applies to a site deployed to a static server (site generated with `nuxt generate`)
 - [ ] Applies to a site deployed to a server with a Node backend

### @nuxtjs/i18n configuration

```
i18n: {
    defaultLocale: "it",
    locales: ["it", "en"],
    vueI18n: {
      legacy: false,
      locale: "it",
      messages: {
        it: {
          welcome: "Benvenuto",
        },
        en: {
          welcome: "Welcome",
        },
      },
    },
  }
```

### Steps to reproduce
Start a new nuxt project, add this package following steps mentioned [here](https://v8.i18n.nuxtjs.org/getting-started/setup), nothing work: 
500
[vite-node] [ERR_INVALID_URL_SCHEME] /.nuxt/i18n.options.mjs

**Comment by kazupon** [comment]
Thank you for reporting!

I've just tried reproduction.
Unfortunately, I could not reproduce this issue:

>vite-node ERR_INVALID_URL_SCHEME /.nuxt/i18n.options.mjs

Could you give us to minimume reproduction repro with github repo or stackbliz please? 🙏

**Comment by m0zz094** [comment]
Here it is https://github.com/m0zz094/i18n-nuxt-demo

**Comment by kazupon** [comment]
Thanks!

I've met the this error:

```
[nuxt] [request error] [unhandled] [500] Cannot read properties of undefined (reading 'path')
  at Module.detectBrowserLanguage (virtual:nuxt:./.nuxt/i18n.internal.mjs:190:70)
  at Module.detectLocale (virtual:nuxt:./.nuxt/i18n.utils.mjs:130:87)
  at ./node_modules/@nuxtjs/i18n/dist/runtime/plugin.mjs:53:45
  at processTicksAndRejections (node:internal/process/task_queues:96:5)
  at async Object.callAsync (./node_modules/unctx/dist/index.mjs:53:16)
  at async applyPlugin (./node_modules/nuxt/dist/app/nuxt.mjs:94:23)
  at async Module.applyPlugins (./node_modules/nuxt/dist/app/nuxt.mjs:104:5)
  at async createNuxtAppServer (./node_modules/nuxt/dist/app/entry.mjs:30:7)
  at async default (./node_modules/@nuxt/vite-builder/dist/runtime/vite-node.mjs:27:18)
  at async Object.renderToString (./node_modules/vue-bundle-renderer/dist/runtime.mjs:172:19)
```

Is the error you saw this error?

**Comment by m0zz094** [comment]
With the beta.1 i got the first error, trying with the beta.2 i got that one you mentioned, btw if i create a index.vue file inside pages the problem is not there so I guess the problem happens only when you're working with app.vue directly

**Comment by kazupon** [comment]
I've fixed this issue.
I'll release this fixing for v8.0.0-beta.3
If you would try to use,  you can do edge version (`@nuxtjs/i18n-edge@8.0.0-beta.2-27781059.fadc56d`)
about how to usage, you can check here: https://github.com/nuxt-community/i18n-module/tree/next#install-edge-version

Thanks!

**Comment by szulcus** [comment]
I have the same problem with edge version. Works fine with `ssr: false`, but but i would like to use ssr in this project. Here is the reproduction: https://gitlab.com/szulcus/easyword.

I'm temporarily hiding my server folder. You can remove the code from `~/pages/index.vue` if needed.

**Comment by kazupon** [comment]
I've just released v8.0.0-beta.3
close
